kubernetes/assets/monitors/monitor_pod_crashloopbackoff.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "[kubernetes] Pod {{pod_name.name}} is CrashloopBackOff on namespace {{kube_namespace.name}}",
 	"type": "query alert",
-	"query": "avg(last_30m):default(avg:kubernetes.containers.state.waiting{reason:crashloopbackoff} by {kube_namespace,pod_name}, 0) >= 1",
+	"query": "max(last_10m):max:kubernetes_state.container.waiting{reason:crashloopbackoff} by {kube_namespace,pod_name} >= 1",
```

Comment:
I wonder if this would get too spammy. A bad deployment with 100 replicas for example would result in 100 alerts.

Reply:
Not sure if there's a way to avoid that (without avalanche control), but I can mention that in the notification message?